chapters/appendix-VIII-SPDX-Lite.md

```diff
@@ -0,0 +1,52 @@
+
+An SPDX Lite file can be used parallel with an SPDX file in software supply chains. 
+
+
+## 2. format of SPDX Lite
+
+The SPDX Lite is the subset of the SPDX specification. The SPDX Lite consists of mandatory part of the Document Creation and Package Information and other basic information. 
+
+The mandatory part of the Document Creation (those are Document Name, SPDX Document Namespace and Creator) is used for keeping the compatibility with SPDX tools.
+
+The main part of the Package Information (those are Package Name,  Package Version, Package File Name, Package Download Location,  Package Home Page, Concluded License, Declared License, Comments on License and Copyright Text) is used for exchanging license information.
+
+In the Package Information, Package SPDX Identifier and Files Analyzed are used for keeping the compatiblity with SPDX tools. 
+
+File Analyzed must be set to "false", when SPDX Lite is used.
+
+Other information (License Identifier, Extracted Text, License Name and License Comment) is used for exchanging license information.
+
+
+## Table of SPDX Lite
+
+| # | corresponding SPDX section no. | License Info. | Tag  | Rationale      |
+|:-----|:----|:--------|:-------|:------------------|
+|L1.1	 |2.4	 |Document Name	| DocumentName |To identify SPDX Lite document	|
+|L1.2	 |2.5	 |SPDX Document Namespace	| DocumentNamespace |To identify SPDX Lite document name	|
+|L1.3	 |2.8	 |Creator	      | Creator |To identify creator	|
+|L2.1	 |3.1	 |Package Name	| PackageName |To identify software	|
+|L2.2	 |3.2	 |Package SPDX Identifier	| SPDXID | To keep compatibility with SPDX. Feedback from SPDX team. |
+|L2.3	 |3.3	 |Package Version	| PackageVersion |To identify specific version	|
+|L2.4	 |3.4	 |Package File Name	| PackageFileName	|To identify specific package|
+|L2.5	 |3.7	 |Package Download Location 	| PackageDownloadLocation |To get the identical software	|
+|L2.6	 |3.8	 |Files Analyzed| FilesAnalyzed | In SPDX Lite, to set "false". To keep compatibility with SPDX. Feedback from SPDX team. |
+|L2.7  |3.11 |Package Home Page	| PackageHomePage	|To verify relevant information|
+|L2.8	 |3.13 |Concluded License	| PackageLicenseConcluded	||
+|L2.9	 |3.15 |Declared License	| PackageLicenseDeclared	||
+|L2.10 |3.16 |Comments on License	| PackageLicenseComments	|To verify additional conditions|
+|L2.11 |3.17 |Copyright Text| PackageCopyrightText	||
+|L3.1	 |6.1	 |License Identifier	| LicenseID	|To specify licenses which are not on the SPDX license list / To specify dual license|
+|L3.2	 |6.2	 |Extracted Text| ExtractedText	|To specify licenses which are not on the SPDX license list / To specify dual license|
+|L3.3	 |6.3	 |License Name	| LicenseName	|To specify licenses which are not on the SPDX license list / To specify dual license|
+|L3.4	 |6.5	 |License Comment| LicenseComment	|To specify licenses which are not on the SPDX license list / To specify dual license|
```

Comment:
Please note that it is also mandatory to have at least one `Relationship` field per Package, if there is more than one Package listed in the SPDX document. Please see [Section 7.1.1 (under the "DESCRIBES" relationship type) and 7.1.2-7.1.5](https://spdx.github.io/spdx-spec/7-relationships-between-SPDX-elements/) for more information on this.

Reply:
Based on work happening in this effort (https://www.ntia.doc.gov/SoftwareTransparency) I would suggest that relationships between elements are so important to a global SBOM system that the Relationship field (7.1, and optionally 7.2) should be added to the SPDX Lite table.

| # | corresponding SPDX section no. | License Info. | Tag  | Rationale      |
|L3.5	 |7.1	 |Relationship| Relationship	|To convey what other elements are included in / To help users collect information about all the elements they are using|